components/charts/types/__tests__/NumberChartCustomizations.test.tsx

```
@@ -0,0 +1,102 @@
+      />
+    );
+
+    expect(screen.getByLabelText('Large')).toBeChecked();
+    expect(screen.getByDisplayValue('Total Users')).toBeInTheDocument();
+    expect(screen.getByLabelText('Decimal Places')).toHaveValue(2);
+  });
+
+  it('should disable all controls when disabled is true', () => {
+    render(<NumberChartCustomizations {...defaultProps} disabled={true} />);
+
+    screen.getAllByRole('radio').forEach((r) => expect(r).toBeDisabled());
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)